Snake/DatabaseLayer/Database.cs

```diff
@@ -1,36 +1,74 @@
-﻿using System.IO;
+﻿using System.Data;
+using System.Data.SqlClient;
+using System.Collections.Generic;
 
 namespace DatabaseLayer
 {
     public class Database
     {
+        private const int MaxNoOfScores = 5;
+        private SqlConnection sqlConnection;
+        private SqlCommand cmd;
+        private SqlDataReader sqlDataReader;
+        private string mazeLevel;
+
+        public Database(string mazeLevel)

```

Comment:
Why do we pass this in to the constructor? I would pass it to each method instead. i.e. getHighSCore(mazeLevel)